src/core/test/base/src/main/scala/TestBase.scala

```diff
@@ -3,19 +3,25 @@
 
 package com.microsoft.ml.spark
 
-import scala.reflect.ClassTag
+import java.nio.file.Files
+
+import com.microsoft.ml.spark.FileUtilities.File
 import org.apache.spark._
 import org.apache.spark.ml._
-import org.apache.spark.sql._
-import org.scalatest._
-import org.apache.spark.sql.DataFrame
-import org.apache.spark.sql.functions._
+import org.apache.spark.ml.util.{MLReadable, MLWritable}
+import org.apache.spark.sql.{DataFrame, _}
 import org.scalactic.source.Position
+import org.scalatest._
+
+import scala.reflect.ClassTag
 
 // Common test tags
 object TestBase {
+
   object Extended extends Tag("com.microsoft.ml.spark.test.tags.extended")
+
   object LinuxOnly extends Tag("com.microsoft.ml.spark.test.tags.linuxonly")
+
 }
 
 trait LinuxOnly extends TestBase {
```

Comment:
This was from a while ago when OpenCV and CNTK tests were linux only. I did not add this though just reformatted the test base file because there were a lot of style errors